script.js

```diff
@@ -0,0 +1,77 @@
+// Write your solutions below

```

Comment:
you declare product.items a few times.

You can just do it once and use it multiple times